templates/template.html

```diff
@@ -166,15 +166,16 @@ <h1 class="title is-size-2">Event Info</h1>
     <h1 class="title is-size-2">What's On</h1>
     <hr class="title-divider"/>
     <div class="content">
-      <p>We hope to run WASD2021 as an in-person event but that may not be possible because of COVID. The following is what we will have available if a physical event can be safely run:</p>
+      <p>Due to the ongoing situation, we can't ensure upcoming events will be run in-person. Our next event, WASD Summer, will be entirely remote, with the below available. We hope to be able to run the next WASD event in person!</p>
```

Comment:
SD: The sentence starting "Our next event..." really should be re-worded IMO; three commas is too much. I also don't quite understand what "with the below available" refers to.

Reply:
Fixed in new commit